servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/NoopAsyncContextProvider.java

```diff
@@ -116,48 +121,50 @@ public ExecutorService wrapJdkExecutorService(final ExecutorService executor) {
     }
 
     @Override
-    public <T> CompletableFuture<T> wrapCompletableFuture(final CompletableFuture<T> future,
-                                                          final AsyncContextMap contextMap) {
-        return future;
+    public ScheduledExecutorService wrapJdkScheduledExecutorService(final ScheduledExecutorService executor) {
+        return executor;
     }
 
     @Override
-    public ScheduledExecutorService wrapJdkScheduledExecutorService(final ScheduledExecutorService executor) {
-        return executor;
+    public <T> CompletableFuture<T> wrapCompletableFuture(final CompletableFuture<T> future,
+                                                          final ContextMap current) {
+        return future;
     }
 
     @Override
-    public Runnable wrapRunnable(final Runnable runnable, final AsyncContextMap contextMap) {
+    public Runnable wrapRunnable(final Runnable runnable, final ContextMap current) {
```

Comment:
I'd still prefer `context` to `current`. Using current almost implies we shouldn't bother to ask for the context and just use the `AsyncContext.context()` current context.

Reply:
Agreed, it was a mix of current/contextMap in the original `AsyncContextProvider`. I aligned them to `current`, but will rename to `context` as you suggested.